src/datagrid/DataGridFilter.tsx

```diff
@@ -62,6 +64,12 @@ export enum FilterType {
     CUSTOM = "Custom",
 }
 
+// Enum for filter position
+export enum FilterPosition {
+    LEFT = "left",
+    MIDDLE = "middle",
```

Comment:
can we rename MIDDLE to CENTER ?? Just a thought

Reply:
Lets call it "Center"

Reply:
done